paper/paper.md

```diff
@@ -86,6 +86,18 @@ where:
 - $\omega$ is the rotor whirl speed;
 - $\mathbf{f}$ is the generalized force vector.
 
+ROSS is capable of evaluating the impact that components such as bearings and seals have on the rotor.
+The numerical models for calculating the dynamic coefficients have relatively high error comparing to experimental
+data [@jolly2018]. Particularly, ROSS' differential is its capability to perform analysis on how the uncertainties
+on these coefficients can impact the rotor dynamic. Thus, through the stochastic analysis performed on ROSS the user
+is able to obtain the probability of failure of a rotor.
```

Comment:
The stochastic part is not implemented yet, so I think it would be better if we remove this part.
Regarding the contribution of the software to the open source community, we can mention that in the rotordynamics are there are no softwares being developed using the open source concept, with the code being fully available on github, issues being tracked online, clear license etc.
Other point is that we are developing the software using python and its scientific ecosystem, which enables the user to also make use this ecosystem, making it easier to run this type of analysis.
It is also easier to integrate the code into other programs, since we only use open source packages and do not depend on proprietary commercial platforms.